integration/system/info_test.go

```diff
@@ -20,52 +20,87 @@ func TestInfoAPI(t *testing.T) {
 	info, err := client.Info(context.Background())
 	assert.NilError(t, err)
 
-	// always shown fields
-	stringsToCheck := []string{
-		"ID",
-		"Containers",
-		"ContainersRunning",
-		"ContainersPaused",
-		"ContainersStopped",
-		"Images",
-		"LoggingDriver",
-		"OperatingSystem",
-		"NCPU",
-		"OSType",
-		"Architecture",
-		"MemTotal",
-		"KernelVersion",
-		"Driver",
-		"ServerVersion",
-		"SecurityOptions",
-	}
-
-	out := fmt.Sprintf("%+v", info)
-	for _, linePrefix := range stringsToCheck {
-		assert.Check(t, is.Contains(out, linePrefix))
-	}
```

Comment:
This one was ... 😂: it literally worked without a daemon and just unmarshaling an empty `system.Info` struct